bestbook/static/build/css/style.css

```diff
@@ -85,9 +85,7 @@ select:focus, select:active, input[type=text]:focus, input[type=text]:active {
 }
 
 .block {
```

Comment:
This class is used in several templates.  It may be better to revert these changes, create a new class with the `border-bottom` rule, and replace the `block` class in `browse.html` with the new class.